File: guides/voice/gather-dtmf-tones-guide/gather-example-step-0/example.5.x.py
from flask import Flask
from twilio import twiml

app = Flask(__name__)


@app.route("/voice", methods=['GET', 'POST'])
def voice():
    """Respond to incoming phone calls with a menu of options"""
    # Start our TwiML response
    resp = twiml.Response()

    # Start our <Gather> verb
    with resp.gather(numDigits=1) as gather:
        gather.say('For sales, press 1. For support, press 2.')

    # If the user doesn't select an option, redirect them into a loop
    resp.redirect('/voice')

    return str(resp)

if __name__ == "__main__":
    app.run(debug=True)

File: guides/voice/gather-dtmf-tones-guide/gather-example-step-2/example.5.x.php
<?php
// Create a route that will handle Twilio webhook requests, sent as an
// HTTP POST to /voice in our application
require_once '/path/to/vendor/autoload.php';

use Twilio\TwiML\VoiceResponse;

// Use the Twilio PHP SDK to build an XML response
$response = new VoiceResponse();

// Use the <Gather> verb to collect user input
$gather = $response->gather(array('numDigits' => 1, 'action' => '/gather'));
// use the <Say> verb to request input from the user
$gather->say('For sales, press 1. For support, press 2.');

// If the user doesn't enter input, loop
$response->redirect('/voice');

// Render the response as XML in reply to the webhook request
header('Content-Type: text/xml');
echo $response;
